python/lsst/sims/catUtils/utils/testUtils.py

```diff
@@ -16,13 +23,296 @@ def calcADUwrapper(sedName=None, magNorm=None, redshift=None, internalAv=None, i
         if internalAv != 0.0 and internalRv != 0.0:
             a_int, b_int = sed.setupCCMab()
             sed.addCCMDust(a_int, b_int, A_v=internalAv, R_v=internalRv)
-    
+
     if redshift is not None and redshift != 0.0:
         sed.redshiftSED(redshift, dimming=True)
-    
+
     a_int, b_int = sed.setupCCMab()
     sed.addCCMDust(a_int, b_int, A_v=galacticAv, R_v=galacticRv)
-    
+
     adu = sed.calcADU(bandpass, photParams=PhotometricParameters())
-    
+
     return adu
+
+
+def makePhoSimTestDB(filename='PhoSimTestDatabase.db', size=1000, seedVal=32, radius=0.1,
+                     displacedRA=None, displacedDec=None,
+                     bandpass='r', m5=None, seeing=None, **kwargs):
+    """
+    Make a test database to storing cartoon information for the test phoSim input
+    catalog to use.
+
+    The method will return an ObservationMetaData object guaranteed to encompass the
+    objects in this database.
+
+    @param [in] filename is a string indicating the name of the DB file to be created
+
+    @param [in] size is the number of objects int he database
+
+    @param [in] seedVal is the seed passed to the random number generator
```

Comment:
The reason is that this code is much older than my knowledge of `numpy.RandomState` ;)  I will update it.